test_libs/pyspec/eth2spec/fuzzing/test_decoder.py

```diff
@@ -9,9 +9,7 @@ def test_decoder():
     rng = Random(123)
 
     # check these types only, Block covers a lot of operation types already.
-    # TODO: Once has Bitlists and Bitvectors, add back
-    #       spec.BeaconState and spec.BeaconBlock
-    for typ in [spec.IndexedAttestation, spec.AttestationDataAndCustodyBit]:
+    for typ in [spec.BeaconState, spec.BeaconBlock]:
```

Comment:
```suggestion
    for typ in [spec.IndexedAttestation, spec.AttestationDataAndCustodyBit, spec.BeaconState, spec.BeaconBlock]:
```

Reply:
`IndexedAttestation ` is part of `AttesterSlashing` in `BeaconBlock. attester_slashings`. Will add `AttestationDataAndCustodyBit`!